### PR TITLE
[compat] Fix asyncio compatibility to support latest python version

### PR DESCRIPTION
**Pull request by Djailla**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [tox]
-envlist = flake8,pypy3,py35,py36
+envlist = flake8,pypy3,py35,py36,py37,py38
 
 [flake8]
 ignore = F811,E226
-exclude = .tox/*,xx/*,__*,docs,*aio.py
```
Comment (wdauchy) [inline tox.ini:6]: is it necessary diff in this commit?

Comment (Djailla) [reply tox.ini:6]: It allows to run CI on the asyncio related files so yes

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.5, 3.6]
+        python-version: [3.5, 3.6, 3.7, 3.8]
```
Comment (dclaisse) [inline .github/workflows/main.yml:11]: This should have been done in another commit.

File: setup.py [hunk 3]
```diff
@@ -74,5 +69,7 @@ def run_tests(self):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```
Comment (dclaisse) [inline setup.py:73]: This should have been done in another commit.

File: tox.ini [hunk 4]
```diff
@@ -1,9 +1,9 @@
 [tox]
-envlist = flake8,pypy3,py35,py36
+envlist = flake8,pypy3,py35,py36,py37,py38
```
Comment (dclaisse) [inline tox.ini:2]: This should have been done in another commit.

File: tox.ini [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 
 [flake8]
 ignore = F811,E226
-exclude = .tox/*,xx/*,__*,docs,*aio.py
+exclude = .tox/*,__*,docs,.venv/*
```
Comment (dclaisse) [inline tox.ini:6]: This should have been done in another commit.

Comment (Djailla) [reply tox.ini:6]: Which part ?
Removinv *aio.py shouldn't ?
